transforms/create-element-to-jsx.js

```diff
@@ -0,0 +1,95 @@
+module.exports = function(file, api) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  const convertObjectExpressionToJSXAttributes = (objectExpression) => {
+    if (!objectExpression.properties) {
+      return [];
+    }
+
+    const attributes = objectExpression.properties.map((property) => {
+
+      if (property.type === 'SpreadProperty') {
+        return j.jsxSpreadAttribute(property.argument);
+      }  else if (property.type === 'Property') {
+        const propertyValueType = property.value.type;
+
+        let value;
+        if (propertyValueType === 'Literal') {
+          value = j.literal(property.value.value);
+        } else if (propertyValueType === 'MemberExpression') {
+          value = j.jsxExpressionContainer(property.value);
+        }
+
+        return j.jsxAttribute(
+          j.jsxIdentifier(property.key.name),
+          value
+        );
+      }
+    });
+
+    return attributes;
+  };
+
+  const convertNodeToJSX = (node) => {
+    if (!node.value || node.value.type !== 'CallExpression') {
+      return undefined;
+    }
+
+    const args = node.value.arguments;
+
+    const elementType = args[0].type;
+    const elementName = elementType === 'Literal' ? args[0].value : args[0].name;
+    const props = args[1];
+
+    const attributes = convertObjectExpressionToJSXAttributes(props);
+
+    const children = node.value.arguments.slice(2).map((child, index) => {
+      if (child.type === 'Literal') {
+        return j.literal(child.value);
+      }
+
+      return convertNodeToJSX(node.get('arguments', index + 2));
+    });
+
+    const openingElement = j.jsxOpeningElement(j.jsxIdentifier(elementName), attributes);
+
+    if (children.length) {
+      return j.jsxElement(
+        openingElement,
+        j.jsxClosingElement(j.jsxIdentifier(elementName)),
+        children
+      );
+    } else {
+      openingElement.selfClosing = true;
+      return j.jsxElement(openingElement);
+    }
+  };
+
+  const update = (node) => {
+    j(node)
+      .replaceWith(
```

Comment:
yeah, you don't need the call to `j`, `replaceWith` is part of every collection so you can just call it directly instead of `forEach`.